nbgrader/docs/source/user_guide/managing_assignment_files.ipynb

```diff
@@ -1008,9 +1013,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python",
+   "display_name": "Python 2",
```

Comment:
I like to avoid committing kernel information to the repository since it changes every time someone edits the notebook. To clear it, can you run:

```
invoke docs
```

from the root of the repository (this will take a little while) and then commit the results?